### DISCUSSION TITLE
Issues while using LipSurf Chrome Extension

### DISCUSSION THREAD

**Discussion by Mister-Curious** [darkreader/darkreader#7307] [General]
Looking for some compatibility testing if possible with a widely used Chrome Extension called [LipSurf](https://www.lipsurf.com/) that adds voice control and other options for Chrome, one of which is an overlay which uses numeric tags. 

When using certain theme engines in Dark Reader the contrasting colors of the tags are quite hard to read, particularly in Dynamic mode (my favorite and only used mode). Here are some screen shots:

https://discuss.lipsurf.com/uploads/default/original/1X/934285970a670eb8593df7119c67b9e51c244da8.jpeg

https://discuss.lipsurf.com/uploads/default/original/1X/922cefd83a2f2706d62b907581ec7a86fedcae2d.jpeg

Wondering if someone could look into this? 

Thanks

**Comment by Gusted** [comment]
Hmm interesting plugin, I'm not able to activate of some sort. As it's trying to request microphone permission, but doesn't seem to request it - chrome doesn't show me the usual window.

So I'm not able to see the numeric tags.

**Reply by Mister-Curious** [reply]
Thanks for the reply. Please let me know if you can get it working, perhaps on another machine, or if someone else is able to investigate.

**Reply by mikob** [reply]
@Gusted hi, I can help you to get it working. Firstly, you need to make sure you're on vanilla Google Chrome (doesn't work on Chromium or other Chromium based browsers). Next, if you right click the extension icon and click options, there you can get it to prompt you for microphone permission. If you don't see the prompt, inside the address bar you will see an icon that can be pressed to manually give permission.

**Comment by mikob** [comment]
LipSurf has this style for the tags:
```
.rnh290318-anno {
    border: 1px solid red;
    display: inline-block;
    z-index: 10;
    background-color: yellow;
    color: #111;
    border-style: dotted;
    padding: 0 1px;
    font-family: Arial, sans-serif;
    border-radius: 6px;
    font-size: 8pt;
    font-weight: bold;
    line-height: 10pt;
    position: absolute;
    box-shadow: #00000081 2px 2px 1px;
}
```

And dark reader applies this:
```
.rnh290318-anno {
border-color: rgb(179, 0, 0);
    background-color: rgb(204, 204, 0);
    color: rgb(221, 218, 214);    /* notice that here is the problem, basically white text on a yellow background, darkreader should do a better job of choosing this color */
    box-shadow: rgb(0 0 0 / 51%) 2px 2px 1px;
}
```

**Reply by mikob** [reply]
related to https://github.com/darkreader/darkreader/issues/7487
